lms/envs/common.py

```diff
@@ -1248,12 +1248,34 @@
 base_application_js = [
     'js/src/utility.js',
     'js/src/logger.js',
+    'js/my_courses_dropdown.js',
+    'js/src/string_utils.js',
+    'js/form.ext.js',
+    'js/src/ie_shim.js',
+    'js/src/accessibility_tools.js',
 ]
 
 dashboard_js = (
     sorted(rooted_glob(PROJECT_ROOT / 'static', 'js/dashboard/**/*.js'))
 )
-discussion_js = sorted(rooted_glob(COMMON_ROOT / 'static', 'coffee/src/discussion/**/*.js'))
+discussion_js = (
+    rooted_glob(PROJECT_ROOT / 'static', 'coffee/src/customwmd.js') +
```

Comment:
Note to reviewers: customwmd sets up some stuff with the Markdown libraries that discussions use.

Discussions also use mathjax rendering, so including those seemed like a good idea.

Previously, everything under coffee/src/... was included via project_js ("application").